### DISCUSSION TITLE
Serverless python not deploying

### DISCUSSION THREAD

**Discussion by Frexuz** [vercel/community#2891] [Help] [answered]
### Page to Investigate

none

### Steps to Reproduce

I'm just using the dashboard to clone the python/hello-world example

it just fails instantly, and the checkly link doesn't even work (no access to this resouce)

![SCR-20230616-ie1](https://github.com/vercel/community/assets/49692/c62c590d-9a08-426c-a068-17fd05c308f0)

looks like a bug on vercel's side?

**Comment by smaeda-ks** [comment, marked as answer]
@Frexuz Sounds like you have Checkly integration installed on your account and it's made available to all projects? Could you disable it or only enable it for projects that actually use it and try again?

**Reply by kg-currenxie** [reply]
wow thanks @smaeda-ks 
that was it :) function is now up and running
